package-overrides/github/nohros/nsPopover@0.6.7.json

```diff
@@ -0,0 +1,16 @@
+{
+  "main": "src/nsPopover",
+  "files": [
+    "src/nsPopover.js",
+    "less/ns-popover.less",
+    "sass/ns-popover.scss",
+  ],
```

Comment:
Ok I can remove the 'files' property, however I was following sample overrides from the registry and many of them include the files override.